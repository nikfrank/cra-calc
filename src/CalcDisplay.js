import React, { Component } from 'react';

class CalcDisplay extends Component {
  render() {
    return (
      <div className="calc-display">
        <p>{this.props.stored}</p>
        <p>{this.props.value}</p>
      </div>
    );
  }
}

export default CalcDisplay;
