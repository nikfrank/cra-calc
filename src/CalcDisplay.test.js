import React from 'react';
import ReactDOM from 'react-dom';
import CalcDisplay from './CalcDisplay';

import { shallow } from 'enzyme';

it('should render the values and func', () => {
  const wrapper = shallow(
    <CalcDisplay value={3} stored={5} func={'%'}/>
  );
  expect(wrapper).toMatchSnapshot();
});
