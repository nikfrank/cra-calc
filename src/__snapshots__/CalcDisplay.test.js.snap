exports[`test should render the values and func 1`] = `
<div
  className="calc-display">
  <p>
    5
  </p>
  <p>
    %
  </p>
  <p>
    3
  </p>
</div>
`;
